src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/group_convolution.cpp

```diff
@@ -1299,6 +1329,31 @@ INSTANTIATE_TEST_SUITE_P(smoke_GroupConv_2D_DW_FP32,
                                             ::testing::Values(empty_plugin_config)),
                          GroupConvolutionLayerCPUTest::getTestCaseName);
 
+const std::vector<std::vector<size_t>> strides2d_Brdgmm = {{1, 1}};
+const std::vector<std::vector<size_t>> dilations2d_Brdgmm = {{1, 1}};
```

Comment:
Why only such strides/dilations are tests? Is it brdgmm limitation?

Reply:
I checked the test cases, there is limitation on dilations https://github.com/openvinotoolkit/oneDNN/blob/c60a9946aa2386890e5c9f5587974facb7624227/src/cpu/x64/jit_brdgmm_dw_conv.cpp#L171, so I updated the test code